l10n_eu_nace/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2019 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    nace_ids = fields.Many2many(
```

Comment:
Just like [`partner_nace` had](https://github.com/Antiun/antiun-odoo-addons/blob/bcc0b3c4762b5a5e04dfbe952a15981ad19c78ee/partner_nace/models/res_partner.py#L31-L35):

```suggestion
    primary_nace_id = fields.Many2one(
        "res.partner.nace",
        string="Primary NACE Activity",
    )
    secondary_nace_ids = fields.Many2many(
```